### DISCUSSION TITLE
Problems accessing demos on local Fore server

### DISCUSSION THREAD

**Discussion by joewiz** [Jinntec/Fore#197] [Q&A] [answered]
I've cloned Fore dev HEAD, and after running `npm i` and `npm run start`, the landing page opens up and looks fine, but when I select the Demos section and choose any of the demos - e.g., http://localhost:8090/demo/getstarted.html - the resulting page doesn't show any of the expected content - no demo snippets. Here's a screenshot, with the Developer Tools showing that the request to http://localhost:8090/demo/node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js returned a 404 error and to http://localhost:8090/demo/demo.js a 500 error:

> <img width="900" alt="Screenshot 2023-02-10 at 13 30 01" src="https://user-images.githubusercontent.com/59118/218170392-34b8fccc-085d-45d6-bc13-97797ec90fbb.png">

I also repeated these steps but with the tag of the latest release (v1.5.0) and had the same results.

I'm using macOS 13.2 on an Intel iMac with nvm running node stable (16.13.2) and Chrome 110.0.5481.77. I'd be happy to provide any other system details that would help.

Does anyone have a suggestion for overcoming these errors?

**Comment by JoernT** [comment, marked as answer]
probably this has to do with missing npm modules in directory demo. Do you see node_modules in your demo dir? If not please try to:
* cd into 'demo'
* `npm i`

**Comment by joewiz** [comment]
@JoernT That worked! Thank you!

**Comment by JoernT** [comment]
has been fixed for the future with a postinstall
